[Docs] Add Volumes and Nodes (#861)
hwameistor/hwameistor: docs/docs/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md

---
sidebar_position: 1
sidebar_label: "LVM Volume"
---

# LVM Volume

> HwameiStor provides LVM-based data volumes. 
> This type of data volume provides read and write performance close to that of native disks, 
> and also offers advanced features such as data volume expansion, migration, high availability, and more.

Reviewer: ```suggestion
HwameiStor provides LVM-based data volumes,
which offer read and write performance comparable to that of native disks.
These data volumes also provide advanced features such as data volume expansion, migration, high availability, and more.
```

The following example will demonstrate how to create a simple non-HA type data volume.

## Steps

### 1. Prepare LVM Storage Node

> It is necessary to ensure that the storage node has available capacity. If there is not enough capacity, 

Reviewer: ```suggestion
Ensure that the storage node has sufficient available capacity. If there is not enough capacity, 
```
